Dashboard/Views/ScrutineeringWindow.axaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Avalonia.Controls;
+using Avalonia.Interactivity;
+using Dashboard.ViewModels;
+
+namespace Dashboard.Views;
+
+public partial class ScrutineeringWindow : Window
```

Comment:
This should be a UserControl and could you rename the name to ScrutineeringView (Both the axeml and axeml.cs) to not cause any conflicts with Avalonia's ViewLocator

```suggestion
public partial class ScrutineeringView : UserControl
```